fog/view/server/src/fog_view_service.rs

```diff
@@ -119,52 +119,61 @@ where
         }
     }
 
+    pub fn create_untrusted_query_response(
```

Comment:
:+1: Like the break out of this and the `query_impl`
too bad GH doesn't seem to handle this diff well, [BeyondCompare](https://www.scootersoftware.com/) does though
![image](https://user-images.githubusercontent.com/101757619/193685353-770f9c36-707a-4b03-b8f1-06b438a8fa36.png)